SU-SOE/soe_profile
# Added SOE specific configs on top of stanford_profile

# READY FOR REVIEW

# Summary
- SOE configs and things on top of stanford_profile
  - The 8.x.2.x branch in the repo I started fresh with the very first commit of this repo's history.
  - I pulled in the history from stanford_profile `8.x-2.x` branch. 
  - I renamed files/namespaces etc. 
  - Did a git checkout of the config directory and fixed anything weird.
  - This should put the soe_profile on a "forked" path from stanford_profile.

# Need Review By (Date)
- 2/4

# Urgency
- high

# Steps to Test
1. use branch in https://github.com/SU-SWS/acsf-lelandd8/pull/61
2. observe the soe site still looks the same with a new install or an existing site updated.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
